src/main/java/com/squareup/kotlinpoet/AnnotationSpec.kt

```diff
@@ -127,59 +101,66 @@ class AnnotationSpec private constructor(builder: AnnotationSpec.Builder) {
   }
 
   class Builder internal constructor(internal val type: TypeName) {
-    internal val members = mutableMapOf<String, MutableList<CodeBlock>>()
+    internal val members = mutableListOf<CodeBlock>()
     internal var useSiteTarget: UseSiteTarget? = null
 
-    fun addMember(name: String, format: String, vararg args: Any) =
-        addMember(name, CodeBlock.of(format, *args))
-
-    fun addMember(name: String, codeBlock: CodeBlock) = apply {
-      members.getOrPut(name, { mutableListOf() }).add(codeBlock)
-    }
+    fun addMember(format: String, vararg args: Any) =
+        addMember(CodeBlock.of(format, *args))
 
-    /**
-     * Delegates to [.addMember], with parameter `format` depending on the given `value` object.
-     * Falls back to `"%L"` literal format if the class of the given `value` object is not
-     * supported.
-     */
-    internal fun addMemberForValue(memberName: String, value: Any) = when (value) {
-      is Class<*> -> addMember(memberName, "%T::class", value)
-      is Enum<*> -> addMember(memberName, "%T.%L", value.javaClass, value.name)
-      is String -> addMember(memberName, "%S", value)
-      is Float -> addMember(memberName, "%Lf", value)
-      is Char -> addMember(memberName, "'%L'", characterLiteralWithoutSingleQuotes(value))
-      else -> addMember(memberName, "%L", value)
+    fun addMember(codeBlock: CodeBlock) = apply {
+      members += codeBlock
     }
 
     fun useSiteTarget(useSiteTarget: UseSiteTarget?) = apply {
       this.useSiteTarget = useSiteTarget
     }
 
     fun build() = AnnotationSpec(this)
+
+    companion object {
+
+      /**
+       * Creates a [CodeBlock] with parameter `format` depending on the given `value` object.
+       * Falls back to `"%L"` literal format if the class of the given `value` object is not
+       * supported.
```

Comment:
not supported –> not known ?

Reply:
In fact it's "is not a special case". I can try to rephrase the doc.